test/image/mocks/zindex-heatmap.json

```
@@ -0,0 +1,22 @@
+{
+"data": [
+    {
+        "type": "scatter",
+        "y": [-0.5,0,1,2,3.5],
+        "x": [-0.5,0,1,2,3.5]
+    }, {
+        "x": [ 3, 2, 1, 0 ],
+        "y": [ 3, 2, 1, 0 ],
+        "z": [
+            [ 98, 97, 96 ],
+            [ 88, 87, 86 ],
+            [ 78, 77, 76 ]
+        ],
+        "type": "heatmap",
+        "showscale": false,
+        "xgap": 5,
+        "ygap": 5,
+        "zindex": 10
+    }
+]
```

Let's provide width and height here: 
```suggestion
 ], { "width": 400, "height": 400" }
```